packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -61,14 +61,22 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid, hideLinks = fa
 
   if (latestVitals && Object.keys(latestVitals)?.length && conceptMetadata?.length) {
     const hasActiveVisit = Boolean(currentVisit?.uuid);
-    const vitalsTakenToday = Boolean(dayjs(latestVitals?.date).isToday());
-    const vitalsOverdue = hasActiveVisit && !vitalsTakenToday;
     const now = dayjs();
-    const vitalsOverdueDayCount = Math.round(dayjs.duration(now.diff(latestVitals?.date)).asDays());
+    const vitalsTakenTimeAgo = dayjs.duration(now.diff(latestVitals?.date));
+    const vitalsThresholdHours = config.vitals.vitalsOverdueThresholdHours;
+    const vitalsOverdue = hasActiveVisit && vitalsTakenTimeAgo.asHours() > vitalsThresholdHours;
```

Comment:
If the number of hours that the vitals were last taken is exactly the number of hours in the threshold, it won't be flagged as overdue. @ibacher @denniskigen Is that expected, or should we change this condition to equal and greater than? If the latter, we should probably change the description of the config too.

Reply:
Equal to or greater than makes sense.

Reply:
Could you effect this change, @dhruvchandak30?